### PR TITLE
[fixes #2182] Add graduation reports *DO NOT MERGE*

### LINKED ISSUE #2182: Add graduation reports for Tracking Protection and No More 404's
This will need to be merged in and pushed to production tonight (a bit behind on timing here =/). Hopefully it won't need much QA since we're just updating an existing YAML field.

### PR DESCRIPTION
**Pull request by ckprice**
- [x] Add Tracking Protection graduation report ([preview page](https://pageshot.net/LBtzvgzEVGsJiyOl/example.com))
- [ ] Add No More 404s graduation report (incoming soon)

@johngruen @pdehaan r?

### CODE REVIEW

File: content-src/experiments/tracking-protection.yaml [hunk 1]
```diff
@@ -91,6 +91,113 @@ created: '2016-09-22T00:07:28.847430Z'
...
+graduation_report: >
+  <h3>Test Pilot Tracking Protection Retirement Report</h3>
+  <p>The Tracking Protection feature in Private Browsing Mode is popular with Firefox users, 
+  but it comes with a few downsides: it can break page layout and multimedia content on 
+  websites, and it sometimes blocks content that isn’t tracking you. We’ve collected qualitative 
+  evidence of this through user feedback, but since we introduced Tracking Protection, 
+  we haven’t been able to quantify the impact of breakage and false positives on the overall 
+  user experience.</p>
+  <p>We launched the Tracking Protection experiment in Test Pilot with a goal of understanding
+   how the block list we use for Tracking Protection in <a target="_blank" href="https://support.mozilla.org/en-US/kb/private-browsing-use-firefox-without-history">Private Browsing Mode</a> and in
```
Comment (pdehaan) [inline content-src/experiments/tracking-protection.yaml:103]: You could remove the /en-US/ in the address and let MDN redirect to the user's most appropriate locale.

File: content-src/experiments/tracking-protection.yaml [hunk 2]
```diff
@@ -91,6 +91,113 @@ created: '2016-09-22T00:07:28.847430Z'
...
+   simple by design: we enabled Tracking Protection and let users disable it on specific sites 
+   or provide reports about websites that seemed to break as a result of the experiment.</p>
+  <p><strong>Here’s how it went down</strong></p>
+  <p>The user interface for the Tracking Protection experiment was simple. When Tracking 
+  Protection was enabled, users saw an icon in their browser’s URL bar. Clicking 
+  the icon opened a panel where participants could report a problem and turn 
+  Tracking Protection off for that domain.</p>
+  <figure>
+  <img src="/static/images/experiments/tracking-protection/experiments_experimentdetail/detail1.jpg" alt="Tracking Protection doorhanger" />
+  </figure>
```
Comment (pdehaan) [inline content-src/experiments/tracking-protection.yaml:115]: Should these figures have an associated `<figcaption>...</figcaption>`?

File: content-src/experiments/tracking-protection.yaml [hunk 3]
```diff
@@ -91,6 +91,113 @@ created: '2016-09-22T00:07:28.847430Z'
...
+  <p>For this experiment, we only collected data when participants turned Tracking 
+  Protection on or off, or when they reported problems with a particular domain. 
+  We did not collect data on other browsing activity and we did not record full URLs.</p>
+  <p><strong>Here’s what we learned</strong></p>
+  <p>We hypothesized that users would frequently disable Tracking Protection on broken sites. 
+  This, it turns out, was not the case. The chart below shows that despite a high 
+  number of participants using Firefox Tracking Protection every day, Tracking 
+  Protection was disabled on very few sites.</p>
+  <p><iframe src="https://docs.google.com/spreadsheets/d/19sMMr2Z6O8Ny_Olisz2sKE13CLIOet9KAlrcn4HSXDk/pubchart?oid=1101471247&amp;format=interactive" width="600" height="380" frameborder="0"></iframe></p>
```
Comment (pdehaan) [inline content-src/experiments/tracking-protection.yaml:128]: Should this `<iframe>` be within the `<figure>...</figure>` block?

File: content-src/experiments/tracking-protection.yaml [hunk 4]
```diff
@@ -91,6 +91,113 @@ created: '2016-09-22T00:07:28.847430Z'
...
+  </figure>
+  <p>Further, the rate at which users disabled Tracking Protection decreased as the 
+  experiment progressed. We think that once users disabled Tracking Protection for 
+  the most problematic sites, they didn’t encounter or weren’t bothered by breakage 
+  to the point that they would disable it for additional sites they visited later in 
+  the experiment.  Since we didn’t measure total page views, there are limits to what 
+  we can conclude about the number of sites broken by Tracking protection. Nevertheless, 
+  this data suggests that breakage may be less widespread than we initially believed, 
+  and it’s worth digging deeper into more specific breakage data.</p>
+  <p><iframe src="https://docs.google.com/spreadsheets/d/19sMMr2Z6O8Ny_Olisz2sKE13CLIOet9KAlrcn4HSXDk/pubchart?oid=206050478&amp;format=interactive" width="600" height="380" frameborder="0"></iframe></p>
```
Comment (pdehaan) [inline content-src/experiments/tracking-protection.yaml:140]: https://validator.w3.org/nu/#textarea says:

> _"The `frameborder` attribute on the `iframe` element is obsolete. Use CSS instead."_

¯\\\_(ツ)\_/¯

Comment (pdehaan) [reply content-src/experiments/tracking-protection.yaml:140]: ultra nit: **eslint-plugin-jsx-a11y** says:

```sh
➜  jsx-a11y-test $ npm test

> jsx-a11y-test@1.0.0 test /Users/pdehaan/dev/tmp/del/jsx-a11y-test
> eslint .

/Users/pdehaan/dev/tmp/del/jsx-a11y-test/index.js
  35:6  error  <iframe> elements must have a unique title property  jsx-a11y/iframe-has-title
  47:6  error  <iframe> elements must have a unique title property  jsx-a11y/iframe-has-title
  79:6  error  <iframe> elements must have a unique title property  jsx-a11y/iframe-has-title

✖ 3 problems (3 errors, 0 warnings)

npm ERR! Test failed.  See above for more details.
```

(Using a hacked empty dir and the following .eslintrc.yml config:)

```yaml
env:
  es6: true
  node: true

extends:
  - eslint:recommended
  - plugin:jsx-a11y/recommended

plugins:
  - jsx-a11y

root: true
```

**Comment by johngruen** [comment]
@clouserw working on this now for No More 404s

File: content-src/experiments/tracking-protection.yaml (hunk 4 above)
Comment (johngruen) [inline content-src/experiments/tracking-protection.yaml]: @pdehaan i'll just add the CSS class

File: content-src/experiments/tracking-protection.yaml (hunk 2 above)
Comment (johngruen) [reply content-src/experiments/tracking-protection.yaml:115]: meh

**Pull request merged**